src/guide/extras/music.md

```diff
@@ -0,0 +1,108 @@
+# Music
+
+aoi.js has an extension named ***"@akarui/aoi.music"*** for music.
+
+## installation
+
+> ```
+> npm i @akarui/aoi.music
+> ```
+
+***for Dev Version***
+
+> ```
+> npm i https://github.com/akaruidevelopment/music#main
+> ```
+
+### setting up in aoi.js
+
+After installing aoi.music, to connect it with aoi.js , we are going to use aoi.js' Voice class.
+
+```js
+const { Voice, LoadCommands, Bot } = require("aoi.js");
+
+const bot = new Bot({
+  token: "DISCORD BOT TOKEN",
+  prefix: ".",
+  intents: ["guilds", "guildMessages", "guildVoiceStates"],
+});
+
+const loader = new LoadCommands(bot);
+
+const voice = new Voice(
+  bot,
+  {
+    cache: {
+      cacheType: "Memory",
+      enabled: true,
+    },
+  },
+  true,
+);
+
+voice.onTrackStart();
+
+loader.load(bot.cmd, "./Commands/commands/"); //bot cmds
+loader.load(voice.cmd, "./Commands/voice/"); //voice cmds
+```
+
+## using aoi.music
+
+as we got a setup in index.js ,now let's create some cmds
+
+### play
+
+
+
```

Comment:
This formatting is need of work,
So many indents for no reason.

Reply:
removed extra space